docs/changelog.rst

```diff
@@ -3,6 +3,9 @@
 
 Changelog
 =========
+- :release:`8.1.0 <16th August 2022>`
+- :support:`124` Updated Discord Invite regex to match leading http, https, www
```

Comment:
Sorry, I lied. We can actually make this link to the area of code changed, by doing the below.
```suggestion
- :support:`124` Updated :obj:`botcore.utils.regex.DISCORD_INVITE` regex to optionally match leading "http[s]" and "www".
```